lib/playground.dart

```diff
@@ -278,17 +281,24 @@ class Playground implements GistContainer, GistController {
     _editpanel.children.first.attributes['flex'] = '';
     editor.resize();
 
-    keys.bind(['ctrl-s'], _handleSave);
-    keys.bind(['ctrl-enter'], _handleRun);
+    keys.bind(['ctrl-s'], sharingDialog.show, "Share");
```

Comment:
Okay, makes sense. But shall I delete this from the menu then until we really do some save action when pressing ctrl-s ?